File: C/divisibility_check.c
#include <stdio.h>

int main()
{
    int a, b;
    printf("Enter two numbers: ");
    if (scanf("%d %d", &a, &b) != 2)
        return 0;

    if (b == 0)
    {
        printf("Cannot divide by zero\n");
        return 0;
    }

    if (a % b == 0)
        printf("%d is divisible by %d\n", a, b);
    else
        printf("%d is not divisible by %d\n", a, b);

    return 0;
}

File: Python/divisibility_check.py
def main():
    a, b = map(int, input("Enter two numbers: ").split())

    if b == 0:
        print("Cannot divide by zero")
        return

    if a % b == 0:
        print(f"{a} is divisible by {b}")
    else:
        print(f"{a} is not divisible by {b}")


if __name__ == "__main__":
    main()

